code/modules/html_interface/map/adv_sec/adv_camera.dm

```diff
@@ -55,6 +55,8 @@ var/global/datum/interactive_map/camera/adv_camera = new
 /datum/interactive_map/camera/show(mob/mob, z, datum/html_interface/currui)
 	z = text2num(z)
 	if (!z) z = mob.z
+	sendResources(mob.client)
```

Comment:
Both of these send resources, we 99.99% know the issue with the map not replacing is a .js not being sent issue and browse_rsc isn't very expensive if you already have the files anyway.

And if it doesn't fix it we'll have another whole angle to consider.